src/main/java/com/royalrangers/service/LoginService.java

```diff
@@ -0,0 +1,24 @@
+package com.royalrangers.service;
+
+import com.royalrangers.bean.ResponseResult;
+import com.royalrangers.enums.Messages;
+import com.royalrangers.model.User;
+import com.royalrangers.utils.ResponseBuilder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LoginService {
+
+    private Messages messages;
+
+    public ResponseResult loginInformation(User user) {
+        if (user.getConfirmed() == false) {
```

Comment:
Please don't ever write like this: if (<some property> == true)...
Your properties has already Boolean type, so you can avoid additional boolean checks and write just: if (<some property>)...